packages/web/src/index.tsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react'
 
 import ReactDOM from 'react-dom'
+// @ts-ignore
```

Comment:
why the ignore? can we add a d.ts file?

Reply:
yeah we def can, just found the declarations file :)